src/main/java/core/basesyntax/MyLinkedListInterface.java

```diff
@@ -1,23 +1,23 @@
-package core.basesyntax;
-
-import java.util.List;
-
-public interface MyLinkedListInterface<T> {
-    void add(T value);
-
-    void add(T value, int index);
-
-    void addAll(List<T> list);
-
-    T get(int index);
-
-    T set(T value, int index);
-
-    T remove(int index);
-
-    boolean remove(T object);
-
-    int size();
-
-    boolean isEmpty();
-}
+package core.basesyntax;

```

Comment:
undo changes to the interface class

Reply:
not fixed
 if the file contents were deleted and then added again seemingly the same, Git might not recognize it as the exact same content due to line-ending or whitespace differences. To compare the differences, you can use Git's diff command: `git diff <commit-hash> -- <file-path>`. Replace <commit-hash> with the hash of the commit before the deletion, and <file-path> with the path to the file.